Documentation/writing-principles-and-guidelines/spelling-and-typo3-terminology.rst

```diff
@@ -1,111 +1,1006 @@
+    -   *List View*
+
+        The :guilabel:`Web -> List` view is a view in the TYPO3 backend used for
+        displaying and managing records in a list format, usually for the
+        File List.
+
+*   M
+
+    ..  _maintenance-mode:
+    -   *Maintenance Mode*
+
+        Maintenance Mode in TYPO3 is used to take the website offline
+        temporarily for updates or maintenance. Only maintainers
+        (administrators) are then able to access the backend and/or frontend.
+
+    ..  _maintenance-tool:
+    -   *Maintenance Tool*
+
+        The Maintenance Tool is a tool in the TYPO3 backend used for performing
+        maintenance tasks and system optimizations. It is part of the "Admin
+        Tools" backend module.
+
+    ..  _makeinstance:
+    -   *makeInstance*
+
+        makeInstance is a method in the TYPO3 PHP API, used for creating
+        instances of classes and objects. It can utilize "Dependency Injection"
+        for service classes.
+
+    ..  _modal:
+    -   *Modal*
+
+        A modal is a dialog or pop-up window in TYPO3 that prompts users to
+        enter or confirm information.
+
+    ..  _module:
+    -   *Module*
+
+        A module is an extendable component in TYPO3 that provides various
+        functionalities and tools. Modules usually come in the form of
+        "Backend Modules", and are part of the left-hand side navigation.
+
+    ..  _multisite:
+    -   *Multisite*
+
+        Multisite refers to the capability of TYPO3 to manage multiple distinct
+        websites from a single installation.
+
+*   O
+
+    ..  _overrides:
+    -   *Overrides*
+
+        Overrides, specifically "TCA Overrides", allow TYPO3 extensions to
+        change central configuration of records and content elements.
+
+*   P
+
+    ..  _package:
+    -   *Package*
+
+        A Package is a bundle of files and resources used for installing and
+        configuring extensions or functionalities in TYPO3. Commonly, TYPO3
+        extensions are provided as "Composer Packages", where this term is the
+        short form of.
+
+    ..  _page-frame---tree-frame---module-frame---navigation-frame:
+    -   *Page Frame / Tree Frame / Module Frame / Navigation Frame*
+
+        Page frame, Tree frame, and Module frame are various sections in the
+        TYPO3 backend used for displaying and navigating content and modules.
+
+    ..  _page-tree:
+    -   *Page Tree*
+
+        The Page Tree is a hierarchical representation of the page structure in
+        the TYPO3 backend used for navigating and managing the website. It is
+        shown in the middle section of the TYPO3 Backend in the places where
+        content is edited.
+
+    ..  _page-view:
+    -   *Page View*
+
+        The Page View is a view in the TYPO3 backend used for editing and
+        managing individual page content.
+
+    ..  _pagebuilder:
+    -   *Pagebuilder*
+
+        Pagebuilders are tools in TYPO3 for creating and designing page layouts
```

Comment:
I have never heard of pagebuilders. Maybe you mean something like: https://www.sitepackagebuilder.com/

Reply:
Heh. Guess I always labelled them like that myself then. I'll rephrase this to "Page or Sitepackage Builder"